src/backend/db/products.js

```diff
@@ -1,4 +1,34 @@
 import { v4 as uuid } from 'uuid';
+import E1 from '../../assets/images/earings/E1.jpg';
+import E2 from '../../assets/images/earings/E2.jpg';
+import E3 from '../../assets/images/earings/E3.jpg';
+import E4 from '../../assets/images/earings/E4.jpg';
+
+import N1 from '../../assets/images/necklace/N1.jpg';
+import N2 from '../../assets/images/necklace/N2.jpg';
+import N3 from '../../assets/images/necklace/N3.jpg';
+import N4 from '../../assets/images/necklace/N4.jpg';
+import N5 from '../../assets/images/necklace/N5.jpg';
+import N6 from '../../assets/images/necklace/N6.jpg';
+import N7 from '../../assets/images/necklace/N7.jpg';
+import N8 from '../../assets/images/necklace/N8.jpg';
```

Comment:
Have an index.js file in your assets folder and export all of your images from there, then it would be easy to write imports in rest of the files.

Reply:
will work on this!